pandas/tests/reshape/merge/test_merge.py

```diff
@@ -1642,6 +1644,57 @@ def test_merge_incompat_dtypes_error(self, df1_vals, df2_vals):
         with pytest.raises(ValueError, match=msg):
             merge(df2, df1, on=["A"])
 
+    @pytest.mark.parametrize(
+        "expected_data, how",
+        [
+            ([1, 2], "outer"),
+            ([], "inner"),
+            ([2], "right"),
+            ([1], "left"),
+        ],
+    )
+    def test_merge_EA_dtype(self, any_numeric_ea_dtype, how, expected_data):
```

Comment:
yep need to change this for 1.3.x

Reply:
So, I should make the change in this PR then, right?

Reply:
yes in this PR itself (as it targets the backport branch)

Reply:
since I did not create this PR, not able to edit it

Reply:
adding this parametrisation definition should resolve this
```
@pytest.mark.parametrize(
    "any_numeric_ea_dtype",
    [
        "UInt8",
        "Int8",
        "UInt16",
        "Int16",
        "UInt32",
        "Int32",
        "UInt64",
        "Int64",
        "Float32",
        "Float64",
    ],
)

Reply:
Isn't this any_nullable_numeric_dtype on 1.3.x?

Reply:
Ah! right, I see it [here](https://github.com/pandas-dev/pandas/blob/363fd402bd24385f4e8c967a60c25a6244ce1693/pandas/conftest.py#L1360)
 But again would need help from maintainer to amend this PR.